r/R/arrow-tabular.R

```diff
@@ -259,3 +259,7 @@ na.omit.ArrowTabular <- function(object, ...) {
 
 #' @export
 na.exclude.ArrowTabular <- na.omit.ArrowTabular
+
+pull.ArrowTabular <- function(x, var = -1) {
```

Comment:
I'd move this to dplyr-collect.R to keep it with the other pull() method definitions

Reply:
Sounds good. Done in https://github.com/apache/arrow/pull/14330/commits/9c8691bb0f430c1e4a0a62003547fa4e57d62d70.